program/src/instructions/edit.rs

```diff
@@ -26,6 +28,7 @@ pub fn process_edit<'info>(
 ) -> Result<()> {
     let list_state = &mut ctx.accounts.list_state;
     list_state.amount = amount;
+    // TODO: ooooof is it a good idea to let them edit currency? what can go wrong?
```

Comment:
Can you think of specific exploits? Not sure if this is different than just closing the listing and re-opening it with a new currency. @richardwu